features/timeseries/src/main/java/org/opennms/netmgt/timeseries/resource/TimeseriesSearcher.java

```diff
@@ -64,18 +65,16 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(TimeseriesSearcher.class);
 
-    private final TimeseriesStorageManager timeseriesStorageManager;
-
     private final TimeSeriesMetaDataDao metaDataDao;
 
-    private final Cache<Tag, Set<Metric>> indexMetricsByTag;
+    private final Cache<TagMatcher, Set<Metric>> indexMetricsByTagMatcher;
```

Comment:
@j-white I used the same cache logic as we did before: Cache by tag matcher instead of cache by tag.
Maybe we want to revisit that, but so far it worked good enough I guess..

Reply:
Makes sense, that can work for now.